Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcContentServer.cs

```diff
@@ -153,97 +156,174 @@ public async Task<GetStatsResponse> GetStatsAsync(GetStatsRequest request, Cance
             };
         }
 
+        private async Task<OpenStreamResult> GetFileStreamAsync (Context context, ContentHash hash)
+        {
+            // For now, take the first store. Change this to interate through all stores
+            // until content is found.
+            IStreamStore store = _contentStoreByCacheName.First().Value as IStreamStore;

```

Comment:
>First [](start = 58, length = 5)

Since you handle differnt items below. is it guarnatted to have at least one contentstore? Else `.FirstOrDefault()?.Value` would be more robust.

Reply:
To resolve this problem, went ahead and implemented next step of iterating through all stores, which naturally handled case of there being none.